easybuild/framework/easyblock.py

```diff
@@ -3657,11 +3657,11 @@ def make_checksum_lines(checksums, indent_level):
 
         checksums_txt = '\n'.join(checksum_lines)
 
-        # if any checksums were provided before, get rid of them
-        if app.cfg['checksums']:
+        # if 'checksums' is specified in easyconfig file, get rid of it (even if it's just an empty list)
+        checksums_regex = re.compile(r'^checksums(?:.|\n)+?\]\s*$', re.M)
```

Comment:
Strictly speaking this regex ^anchor is probably wrong. I think I can write a fully correct EC where each line starts with a blank...

Reply:
No, you can't do that. I just tried, starting each line with a single space, and then you get `Parsing easyconfig file failed: unexpected indent`.

Reply:
ok